### ISSUE TITLE
Unable to install React 19 due to react-dom dependency

### ISSUE THREAD

**Issue by ugnius-s** [barvian/number-flow#85]
### NumberFlow version

@number-flow/react@0.4.2

### Framework version

react@19

### Describe the bug and the steps to reproduce it

Hello. I'm receiving a dependency error when trying to upgrade react from 18 to 19.

```
npm error node_modules/react-dom
npm error   peer react-dom@"^18" from @number-flow/react@0.4.2
npm error   node_modules/@number-flow/react
npm error     @number-flow/react@"^0.4.2" from the root project
npm error
```

Would greatly appreciate if we can bump the version.

**Comment by barvian** [comment]
Hi, I just released 0.4.3 with bumped React peer deps following 19's release. Please comment if you're still experiencing issues 👋

**Comment by ugnius-s** [comment]
Works well. Thank you
